diesel_compile_tests/tests/compile-fail/right_side_of_left_join_requires_nullable.rs

```diff
@@ -27,13 +27,17 @@ fn main() {
     let join = users::table.left_outer_join(posts::table);
 
     // Invalid, only Nullable<title> is selectable
-    let _ = join.select(posts::title); //~ ERROR E0277
+    let _ = join.select(posts::title);
```

Comment:
Yep, it's one of those errors we should cover in the docs. Or, [you know](https://deterministic.space/hook-into-rustc-errors.html) customize.